web/packages/teleterm/src/ui/AppUpdater/AutoUpdatesManagement.tsx

```diff
@@ -174,6 +164,8 @@ function makeOptions({
     value: '',
   };
 
+  const getClusterName = (uri: RootClusterUri) => routing.parseClusterName(uri);
```

Comment:
`parseClusterName` is a bit of a cursed name at this point because it returns two different things depending on the kind of URI that is passed to it.

At minimum I think we should update the JSDoc of `parseClusterName`. The part about using it only when `ClustersService` cannot  be used is no longer true I guess.

I don't think we'll find a better name for it. It might be doing too many things at this point? Like maybe long term it'd be a good idea to have `parseProfileName` and `parseLeafClusterName`. I'd have to look through this PR again, but I don't think there's a context in which we call `parseClusterName` with some kind of URI (we don't know which kind) and are happy to use either result.